tests/unit/async_/io/test_class_bolt.py

```diff
@@ -65,7 +65,8 @@ def test_class_method_protocol_handlers():
         ((5, 1), 1),
         ((5, 2), 1),
         ((5, 3), 1),
-        ((5, 4), 0),
+        ((5, 4), 1),
+        ((5, 5), 0),
```

Comment:
How come we need 5.5 here? Is this just future-proofing?

Reply:
It asserts that there are `0` Bolt 5.5 handlers registered, i.e., the driver does *not* support Bolt 5.5.

Reply:
Ah ha that makes more sense, thanks.